src/entity/Users/OrganizerFactory.java

```diff
@@ -7,6 +7,6 @@
 
 public interface OrganizerFactory {
     Organizer create(String username, String password, Event eventOf, ArrayList<Event> joinedEvents,
```

Comment:
I'm not sure if `create() `needs this many attributes. Most of the attributes here, `username`, `password`, `joinedEvents`, are already carried by a `CommonUser`. So making it to become a `CommonOrganizer`, the only additional piece of the information we need is `eventOf`. Because `CommonOrganizer` extends `CommonUser`, it will inherit all instance variables from `CommonUser`, along with the setters and getters for them. For now, I suggest reduce the parameters of this method, given that it actually doesn't need to do that much job. I could be wrong! We can discuss on this further.